app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
```

Comment:
- [ ] @AlselwiAisha Great work on your comments controller, kindly consider removing these two lines as they don't accomplish any function, it's good practice to have only functional code in the codebase. kindly implement the same in your post-controller